lib/src/models/excluded_identifiers_parameter.dart

```diff
@@ -1,22 +1,22 @@
-/// Model class for AvoidReturningWidgetsExclude parameters
-class AvoidReturningWidgetsExclude {
+/// Model class for ExcludeRule parameters
+class ExcludedIdentifiersParameter {
```

Comment:
This is a single identifier to be excluded, use singular form.
```suggestion
class ExcludedIdentifierParameter {
```